app/models/concerns/thredded/post_common.rb

```diff
@@ -54,9 +54,31 @@ def context_options
       {
         asset_root: Rails.application.config.action_controller.asset_host || '',
         post:       self,
+        whitelist:  sanitize_whitelist,
       }
     end
 
+    def sanitize_whitelist
```

Comment:
We probably should define our own constant, because currently:
a) It modifies a constant in another module, dangerous (the user may be using the `SanitizationFilter::WHITELIST` for something else).
b) It evaluates this for every post.